cachito/workers/config.py

```diff
@@ -50,6 +50,8 @@ class Config(object):
     worker_prefetch_multiplier = 1
     # Configurable batch size of the json payload so that the cachito API request doesn't time out
     cachito_deps_patch_batch_size = 50
+    # Configurable number of days before which a request becomes stale
+    cachito_request_lifetime = 1
```

Comment:
Optional: It'd be nice if these were alphabetical like the rest of the settings. It just makes it easier to find things.